gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/BaseEntity.java

```diff
@@ -377,21 +376,21 @@ public Blob getBlob(String name) {
   @Override
   protected Object fromPb(byte[] bytesPb) throws InvalidProtocolBufferException {
     Builder<?, ?> builder = emptyBuilder();
-    builder.fill(DatastoreV1.Entity.parseFrom(bytesPb));
+    builder.fill(com.google.datastore.v1beta3.Entity.parseFrom(bytesPb));
     return builder.build();
   }
 
   protected abstract Builder<?, ?> emptyBuilder();
 
   @Override
-  protected final DatastoreV1.Entity toPb() {
-    DatastoreV1.Entity.Builder entityPb = DatastoreV1.Entity.newBuilder();
+  protected final com.google.datastore.v1beta3.Entity toPb() {
+    com.google.datastore.v1beta3.Entity.Builder entityPb = com.google.datastore.v1beta3.Entity.newBuilder();
+    Map<String, com.google.datastore.v1beta3.Value> propertiesPb = new HashMap<String, com.google.datastore.v1beta3.Value>();
```

Comment:
An alternative would be to call entityPb.getMutableProperties() and add directly.